scripts/force-lowest-dependencies.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * This file is part of the Symfony Webpack Encore package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const fs = require('fs');
+const childProcess = require('child_process');
+
+function getLowestVersion(dependency, range) {
+    return new Promise((resolve, reject) => {
+        childProcess.exec(
+            `npm view "${dependency}@${range}" version`,
+            { encoding: 'utf-8' },
+            (error, stdout) => {
+                if (error) {
+                    reject(`Could not retrieve versions list for "${dependency}@${range}"`);
+                }
+
+                const versions = stdout
+                    .split('\n')
+                    .filter(line => line);
+
+                if (versions.length === 0) {
+                    reject(`Could not find a lowest version for "${dependency}@${range}"`);
+                }
+
+                const parts = versions[0].split(' ');
+
+                // If there is only one version available that version
+                // is directly printed as the output of npm view.
+                if (parts.length === 1) {
+                    resolve([dependency, parts[0]]);
+                }
+
+                // If multiple versions are available then it outputs
+                // multiple lines matching the following format:
+                // <package>@<version> '<version>'
+                if (parts.length === 2) {
+                    resolve([dependency, parts[1].replace(/'/g, '')]);
+                }
```

Comment:
Yeah, I wasn't really happy with that part... I guess calling the npm.js API directly would have been a cleaner way to do it but then we would also have had to filter the versions ourselves (and since it happens before `npm install`/`yarn`, we don't have access to the `semver` package at that point).

Thank you for highlighting that part by the way, I noticed that I forgot some `return` in promises... not a big issue since a promise won't be able to change its state after being resolved/rejected but still something that had to be fixed :)